### PR TITLE
Add docs for library integrations

### PR DESCRIPTION
**Pull request by osanseviero**
Add documentation for library integrations!

Two steps missing

- [ ] Update images for higher-quality (will be done in follow-up)
- [ ] Incorporate into docs-revamp branch structure

### CODE REVIEW

**Comment by merveenoyan** [comment]
I will add mine tomorrow.

**Comment by simoninithomas** [comment]
(From slack) Late to the party I can't update the doc now, it's not just copy and paste there's a reset/cleaning I need to do with the documentation given the update of Antonin and VecNorm. If it can't wait for all the hub docs you've updated we can merge your hub doc PR and I'll open a new one PR for tomorrow morning?

**Comment by osanseviero** [comment]
Hey League of Open Source! This PR adds initial documentation for libraries with integration with Hugging Face. This is an initial version and will be launched with new docs, but we can iterate on it.

* :hugs: transformers: cc @LysandreJik @sgugger 
* Adapter Transformers: cc @JoPfeiff @calpt 
* AllenNLP: cc @epwalsh @dirkgr 
* Asteroid: cc @TParcollet 
* ESPnet: cc @sw005320 @kan-bayashi
* Fastai: cc @omarespejel @jph00 @muellerzr  
* Keras: cc @merveenoyan  @fchollet @ariG23498  
* Sentence Transformers: cc @nreimers 
* SpaCy: cc @ines @adrianeboyd 
* SpeechBrain cc @TParcollet
* Stanza: cc @AngledLuffa @merveenoyan 
* TensorBoard: cc @severo 

Also cc @adrinjalali @julien-c

File: docs/hub/stanza.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+All models on the Hub come up with the following features:
+1. An automatically generated model card with a brief description and metadata tags that help for discoverability.
+2. An interactive widget you can use to play out with the model directly in the browser (for named entity recognition and part of speech).
+3. An Inference API that allows to make inference requests (for named entity recognition and part of speech).
+
+
+## Using existing models
+
+The `stanza` library automatically downloads models from the Hub. You can use `stanza.download` to download the model from the Hub and then `stanza.Pipeline` to do inference.
```
Comment (AngledLuffa) [inline docs/hub/stanza.md:21]: the latest version doesn't even need `stanza.download` - we are working on adopting the automatically download paradigm other packages use

File: docs/hub/stanza.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+stanza.download('en') # download English model
+nlp = stanza.Pipeline('en') # initialize English neural pipeline
+doc = nlp("Barack Obama was born in Hawaii.") # run annotation over a sentence
+```
+
+
+## Sharing your models
+
+To add new official Stanza models, you can follow the process to [add a new language](https://stanfordnlp.github.io/stanza/new_language.html). You can also find the official script to upload models to the Hub [here](https://github.com/stanfordnlp/huggingface-models/blob/main/hugging_stanza.py).
```
Comment (AngledLuffa) [inline docs/hub/stanza.md:34]: i would encourage people to send the models back to stanford first.  i suppose leaving that comment about the upload script there will inform future people how to redo the models if i leave stanford w/o sufficient documentation

File: docs/hub/stanza.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+
+## Using existing models
+
+The `stanza` library automatically downloads models from the Hub. You can use `stanza.download` to download the model from the Hub and then `stanza.Pipeline` to do inference.
+
+```python
+import stanza
+
+stanza.download('en') # download English model
```
Comment (AngledLuffa) [inline docs/hub/stanza.md:26]: specifically, the call to `download` here is not necessary any more

**Comment by AngledLuffa** [comment]
Looks great, thanks!  I left a couple small comments, as our latest version simplifies the interface some.

We also have CoreNLP (the Java package) hosted on HF:

https://huggingface.co/stanfordnlp/CoreNLP

https://huggingface.co/stanfordnlp/corenlp-english-default  and other model packages

**Review by NimaBoscarino — APPROVED** [review]

**Pull request merged**